src/Registry/registry_instance.jl

```diff
@@ -82,8 +82,7 @@ function uncompress(compressed::Dict{VersionRange, Dict{String, T}}, vsorted::Ve
             uv = uncompressed[v]
             for (key, value) in data
                 if haskey(uv, key)
-                    # Change to an error?
-                    error("Overlapping ranges for $(key) in $(repr(path)) for version $v.")
+                    error("Overlapping ranges for $(key) in manifest for version $v.")
```

Comment:
```suggestion
                    error("Overlapping ranges for $(key) for version $v in registry.")
```

is probably the best we can do here without refactoring things a bit so that this function has access to more information